prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -145,13 +145,25 @@ static int tcpx_ep_enable_xfers(struct tcpx_ep *ep)
 				      ep->sock, POLLIN, tcpx_try_func,
 				      (void *) &ep->util_ep,
 				      &ep->util_ep.ep_fid.fid);
+		if (ret)
+		{
```

Comment:
nit: Please place { on same line as if statement.  Comment applies throughout this patch.

Reply:
Oh, again, sorry. I guess it would help to use checkpatch. Seems to be the linux coding style, maybe we can borough checkpatch.pl from it?